README.md

```diff
@@ -4,6 +4,8 @@ The home repository for the (Re)usable Data Project. Please see:
 
 http://reusabledata.org
 
+This is a volunteer and community driven effort. As such, we appreciate your efforts to self-curate and make a pull request. As well, if you are interested in adding a new resource, updating a current resource, or have some other issue, we would ask you to create an issue in our tracker and we will do our best to make updates and provide feedback in a timely manner.
```

Comment:
I prefer removing both "As such," and "As well,", but the message content is sound.

Reply:
Changes made.